src/libmain.c

```diff
@@ -406,7 +406,7 @@ static gint get_windows_socket_port(void)
 	if (! g_file_test(configfile, G_FILE_TEST_IS_REGULAR))
```

Comment:
This code could actually be simplified by basically having the same code path for the with and without config cases: just don't load the config if it doesn't exist, and then `utils_get_setting_integer()` will return the default.  But that's not for a point release.

Reply:
Done in 58c5e34a3b3763bd0f22152a6df02db68bd7c8ab